src/test/scala/edu/ie3/mobsim/MobilitySimulatorTestData.scala

```diff
@@ -0,0 +1,116 @@
+  CHARGING_HUB_HIGHWAY,
+  CHARGING_HUB_TOWN,
+  WORK
+}
+import edu.ie3.mobsim.model.{ChargingBehaviorTestData, ElectricVehicle}
+import edu.ie3.simona.api.data.ev.ExtEvData
+import edu.ie3.simona.api.data.ev.ontology.builder.EvMovementsMessageBuilder
+
+import java.util.UUID
+import scala.collection.immutable.{SortedSet, TreeSet}
+
+trait MobilitySimulatorTestData extends ChargingBehaviorTestData {
+  val evData: ExtEvData = new ExtEvData(ActorRef.noSender, ActorRef.noSender)
+  val builder = new EvMovementsMessageBuilder
+
+  def electricVehicles(seq: Seq[ElectricVehicle]): SortedSet[ElectricVehicle] =
+    SortedSet.empty[ElectricVehicle] ++ seq.toSet
+
+  def evIsParking(evs: Seq[ElectricVehicle]): SortedSet[ElectricVehicle] = {
+    evs.foreach { ev =>
+      ev.copyWith(
+        storedEnergy = zero,
+        destinationPoiType = CHARGING_HUB_HIGHWAY,
+        destinationCategoricalLocation =
+          charging_hub_highwayPoi.categoricalLocation,
+        destinationPoi = charging_hub_highwayPoi,
+        parkingTimeStart = givenSimulationStart,
+        departureTime = givenSimulationStart.plusHours(5)
+      )
+    }
+
+    TreeSet.empty[ElectricVehicle] ++ evs.toSet
+  }
+
+  def evIsDeparting(evs: Seq[ElectricVehicle]): SortedSet[ElectricVehicle] = {
+    evs.foreach { ev =>
+      ev.copyWith(
+        storedEnergy = half,
+        destinationPoiType = WORK,
+        destinationCategoricalLocation = workPoi.categoricalLocation,
+        destinationPoi = workPoi,
+        parkingTimeStart = givenSimulationStart.plusHours(-4),
+        departureTime = givenSimulationStart
+      )
+
+      ev.setChargingAtSimona(true)
+      ev.setChosenChargingStation(Some(cs6.getUuid))
+    }
+
+    TreeSet.empty[ElectricVehicle] ++ evs.toSet
+  }
+
+  protected val chargingPointsAllTaken: Map[UUID, Integer] = {
+    Map(cs6.getUuid -> Integer.valueOf(0))
+  }
+
+  protected val chargingPointsAllFree: Map[UUID, Integer] = {
+    Map(cs6.getUuid -> cs6.getChargingPoints)
+  }
+
+  protected val pricesAtChargingStation: Map[UUID, Double] = {
+    Map(cs6.getUuid -> 0.0)
+  }
+
+  protected val arrivingEv: ElectricVehicle = {
+    ev1
+    ev1.copy(
+      destinationPoi = charging_hub_highwayPoi,
+      destinationPoiType = CHARGING_HUB_HIGHWAY,
+      destinationCategoricalLocation =
+        charging_hub_highwayPoi.categoricalLocation
+    )
+  }
+
+  protected val evChargingAtSimonaWithStation: ElectricVehicle = {
+    ev1
+    ev1.copy(chargingAtSimona = true)
+    ev1.copy(chosenChargingStation = Some(cs6.getUuid))
+  }
+
+  protected val evChargingAtSimonaWithoutStation: ElectricVehicle = {
+    ev1
+    ev1.copy(chargingAtSimona = true)
+  }
+
+  val mobSim: MobilitySimulator = new MobilitySimulator(
+    evData = evData,
+    chargingStations = chargingStations,
+    poisWithSizes = poisWithSizes,
+    startTime = givenSimulationStart,
+    electricVehicles = electricVehicles(Seq(ev1, ev2, ev3, ev4, ev5)),
```

Comment:
... this is everything you need:

```suggestion
    electricVehicles = SortedSet(ev1, ev2, ev3, ev4, ev5),
```